ui/components/multichain/pages/connections/connections.tsx

```diff
@@ -85,10 +87,9 @@ export const Connections = () => {
     setShowDisconnectedAllAccountsUpdatedToast,
   ] = useState(false);
 
-  const urlParams: { origin: string } = useParams();
-  const securedOrigin = decodeURIComponent(urlParams.origin);
+  const { origin } = useParams<Params>() as Params;
```

Comment:
This is resulting in a lint error (a variable named `origin` is already declared)

Reply:
Oh whoops!  Put it back to `urlParams.origin`